src/overlay/OverlayManager.h

```diff
@@ -183,6 +183,10 @@ class OverlayManager
     // Return the persistent overlay metrics structure.
     virtual OverlayMetrics& getOverlayMetrics() = 0;
 
+    // Ensure any metrics that are "current state" gauge-like counters reflect
+    // the current reality as best as possible.
+    virtual void syncMetrics() = 0;
```

Comment:
Is this called anywhere? Should it be called in `ApplicationImpl::syncAllMetrics`?